src/coin/skycoin/models/sky_test.go

```diff
@@ -1,188 +0,0 @@
-package skycoin
```

Comment:
Functions in this file have to remain separate from he rest , since they've been copied verbatim from Skycoin code base .